backend-shared/util/errors/error.go

```diff
@@ -0,0 +1,82 @@
+package errors
```

Comment:
Let's change the package from `errors` to `gitopsErrors`, so that it doesn't conflict with the existing built-in `errors` packaged, as noted by Panos in his JIRA story comment.

You will likely also need to update the other code that references this package, and rename the parent folder of this file.